db/migrate/20150421191213_add_column_to_company.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnToCompany < ActiveRecord::Migration
+  def change
+    add_column :companies, :hiring, :boolean
```

Comment:
I just wanted to be clear that the hiring status added here is different than the status we put in the CMS. This one will reflect whether or not the company has indicated that they are hiring, whereas the other status you mention (which we can certainly add later) will reflect whether the company has been contacted.